spec/modules-spec.js

```diff
@@ -129,3 +131,17 @@ describe('Module._nodeModulePaths', function () {
     })
   })
 })
+
+describe('require', () => {
+  describe('when loaded URL is not file: protocol', () => {
+    it('searches for module under app directory', async () => {
+      const w = new BrowserWindow({
+        show: false
+      })
+      w.loadURL('about:blank')
+      const result = await w.webContents.executeJavaScript('typeof require("q").when')
+      assert.equal(result, 'function')
+      w.destroy()
```

Comment:
This should be done in an `afterEach` block in case the assert fails, so the window does not leak across tests.